### PR TITLE
feat: implement remove liquidity page revamp KEP-62

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/components/Header/groups/SwapNavGroup.tsx [hunk 1]
```diff
@@ -120,9 +120,8 @@ const SwapNavGroup = () => {
                   <StyledBridgeIcon height={15} />
                 </IconWrapper>
                 <Flex alignItems={'center'} sx={{ flex: 1 }} justifyContent={'space-between'}>
-                  <Text>
-                    <Trans>Bridge</Trans>
-                  </Text>
+                  <Trans>Bridge</Trans>
```
Comment (namgold) [inline src/components/Header/groups/SwapNavGroup.tsx:123]: can you help explain why remove <Text>

File: src/utils/sendTransaction.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ export async function sendEVMTransaction(
   encodedData: string,
   value: BigNumber,
   handler?: (response: TransactionResponse) => void,
+  chainId?: number,
```
Comment (nguyenhuudungz) [inline src/utils/sendTransaction.ts:23]: `ChainId`?

File: src/utils/index.ts [hunk 3]
```diff
@@ -72,9 +72,10 @@ export function shortenAddress(chainId: ChainId, address: string, chars = 4): st
  * @param value BigNumber
  * @returns BigNumber
  */
-export function calculateGasMargin(value: BigNumber): BigNumber {
+export function calculateGasMargin(value: BigNumber, chainId?: number): BigNumber {
```
Comment (nguyenhuudungz) [inline src/utils/index.ts:75]: `ChainId` instead of `number`?

File: src/components/TransactionConfirmationModal/index.tsx [hunk 4]
```diff
@@ -236,7 +236,7 @@ const StyledAlert = styled(Alert)`
 `
 export function TransactionErrorContent({ message, onDismiss }: { message: string; onDismiss: () => void }) {
   const theme = useTheme()
-  const [showDetail, setShowDetail] = useState<boolean>(false)
+  const [showDetail, setShowDetail] = useState<boolean>(true)
```
Comment (nguyenhuudungz) [inline src/components/TransactionConfirmationModal/index.tsx:239]: Remove boolean (optional).

**Review by nguyenhuudungz — APPROVED** [review]

File: src/pages/RemoveLiquidityProAmm/styled.tsx [hunk 5]
```diff
@@ -1,53 +1,77 @@
...
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    padding: 0;
+    background: transparent;
+  `}
+`
+
+export const TokenId = styled.div<{ color: string }>`
+  font-size: 1rem;
+  font-weight: 500;
+  color: ${({ color }) => color};
```
Comment (namgold) [inline src/pages/RemoveLiquidityProAmm/styled.tsx:66]:
```suggestion
  ${({ color }) => `color: ${color}`};
```

File: src/components/Header/groups/SwapNavGroup.tsx (hunk 1 above)
Comment (viet-nv) [reply src/components/Header/groups/SwapNavGroup.tsx:123]: redundant div tag.

Comment (namgold) [reply src/components/Header/groups/SwapNavGroup.tsx:123]: <Text as='span'> can help? Just curious, not required to add it back.

File: src/utils/sendTransaction.ts (hunk 2 above)
Comment (viet-nv) [reply src/utils/sendTransaction.ts]: fixed

File: src/utils/index.ts (hunk 3 above)
Comment (viet-nv) [reply src/utils/index.ts]: fixed

File: src/components/TransactionConfirmationModal/index.tsx (hunk 4 above)
Comment (viet-nv) [reply src/components/TransactionConfirmationModal/index.tsx:239]: explicit is alway better than implicit =))

**Review by nguyenhuudungz — APPROVED** [review]

**Review by XiaoYhun — APPROVED** [review]

**Pull request merged**